src/routes/model-registry/index.tsx

```diff
@@ -86,6 +94,20 @@ export const ModelRegistry = () => {
                                         </TableCell>
                                         <TableCell>{model.createdBy}</TableCell>
                                         <TableCell>{model.dateModified}</TableCell>
+                                        <TableCell>
+                                            <Dropdown
+                                                menuButton={
+                                                    <EllipsisHorizontalIcon className='h-6 w-6' />
+                                                }
+                                            >
+                                                <DropdownItems>
+                                                    <DropdownItem>Edit</DropdownItem>
+                                                    <DropdownItem>Archive</DropdownItem>
+                                                    <Divider />
+                                                    <DropdownItem>Delete</DropdownItem>
```

Comment:
Delete isn't a concept we really have here

Reply:
I saw that it wasn't in the graphql methods, but figured it might be in the future.

So it's just edit, archive, and publish?

Reply:
Yeah, for auditability we never want to take irrevocably destructive actions. If someone ever needs to go back and answer "What version of what model was deployed and serving inferences at this point in time" to satisfy any sort of regulatory, legal, or other request, it's no bueno for the answer to be, "Oops, we deleted the parent model so we have no idea"

Reply:
Ah. That makes sense.

I'll go ahead and remove that delete functionality then.